<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imgSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="procBtn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 로드</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgDst">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>원본 이미지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>결과 이미지</string>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>710</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>706</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>임계값 :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboThr">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>707</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>707</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>이진화방법</string>
   </property>
  </widget>
  <widget class="QPushButton" name="circleBtn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>원찾기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
